network/load-balancer/reference-content/configuring-backends.mdx

```diff
@@ -0,0 +1,132 @@
+
+When activated, sticky sessions bind a user's session to a specific server in the pool of backend servers. This ensures that all subsequent sessions from the user are sent to the same backend server while there is at least one active session. Sticky sessions can be **cookie based** or **IP based** (aka table-based).
+
+- **IP-based** stickiness uses the source (client) IP address to stick sessions to backends. This is the only possible method for TCP backends.
+- **Cookie-based** stickiness uses an HTTP cookie to identify a session, and the server in the backend pool that it will stick to. For each incoming request without a stickiness cookie, the Load Balancer creates a cookie and selects a backend for the session. Subsequent requests providing this cookie will land on the same backend. You can customize the name of the cookie.
+
+## Configuring advanced settings
+
+Recommended settings for all the parameters below are selected by default, however you can modify them if you wish.
+
+### Timeout
+
+Timeout settings allow you to define the maximum time that backend servers should be given to establish connections and to process requests. The following timeout settings are available:
+
+- **Tunnel timeout**: Defines the maximum allowed tunnel inactivity time (in ms) between a client and a backend server after a Websocket is established. This value takes precedence over client and server timeout in determining when to close the connection. The default value is 900 000, the minimum value is 0 and the maximum value is 2 147 483 647.
+- **Server timeout**: Defines the maximum allowed time (in ms) that a backend server has to process a request. The default value is 300 000, the minimum value is 0 and the maximum value is 2 147 483 647.
+- **Connection timeout**: Defines the maximum allowed time (in ms) to establish a connection between a client and a backend server. The default value is 5 000, the minimum value is O and the maximum value is 2 147 483 647.
+
+### Backend protection
+
+Backend protection settings define when the Load Balancer should view a backend server as having reached its maximum number of simultaneous connections / requests, and how to handle sticky sessions in this context.
+
+- **Limit backend load**: When this toggle is deactivated, the default values for backend protection apply. Activate the toggle to modify the following values:
+- **Max simultaneous**: Defines the maximum number of simultaneous requests (for HTTP) or simultaneous connections (for TCP) to a backend. This is particularly relevant when using the [First available](#balancing-method) balancing method. The default value is 3, the minimum value is 1, and the maximum value depends on the Load Balancer type.
+- **Queue timeout**: Defines the maximum length of time (in ms) to queue a request/connection for a given backend if [stickiness](#sticky-sessions) is enabled. The default value is 5 000, the minimum value is 1 and the maximum value is 2 147 483 647.
```

Comment:
@RoRoJ, here the setting is again defined at the backend level but applies to each backend server in the backend individually.
Do you plan on adding an explanation about the feature mechanism in the documentation?
If stickiness is disabled, we skip backend servers that have reached their connection limit and try others that still have connection slots available. If stickiness is enabled we then queue the request/connection into a queue dedicated to the selected backend server and wait for an available connection slot until the timeout is reached.

Reply:
I elaborated with the extra info, thanks